docs/backend/guides/understanding-milmove-routing.md

```diff
@@ -0,0 +1,115 @@
+# Understanding MilMove Routing
+
+This document is a brief summary on how routing on MilMove works.
+
+## Routing
+
+The routing is initialized in `serve.go`, which calls `InitRouting` inside of `routing_init.go`. This initializes the routing that will be used by the MilMove application. This sets up routing for all 3 sites, the MilMove application, office, and admin. Perhaps this not ideal, but this is how it works as of the writing of this document (December 2022).
+MilMove is using [Gorilla Mux](https://github.com/gorilla/mux) for routing.
+
+## Code Snippet Analysis
+
+:::note
+This document won't analyze all the code related to how server side routing works, but select ones that are considered notable and for which there is extensive knowlege on.
```

Comment:
**nitpick:** the last "on" in the sentence is not needed. #GrammarNerd

Reply:
I appreciate the grammar nerd!